src/legacy/local_server/server_admin/c_comdef_admin_ajax_handler.class.php

```diff
@@ -1499,9 +1499,11 @@ public function TranslateToJSON(
                     $row = substr($row, 0, strlen($row) - 1);
                 }
                 $row = explode('","', $row);
-                foreach ($row as $column) {
-                    if (isset($column)) {
-                        $line[$keys[$index++]] = $column;
+                if (is_array($row)) {
+                    foreach ($row as $column) {
+                        if (isset($column)) {
+                            $line[$keys[$index++]] = $column;
+                        }
                     }
                 }
                 array_push($temp_keyed_array, $line);
```

Comment:
Does this need to be in the id condition too?

Reply:
yea it prob should be, otherwise its potentially just doing an array push on null for no reason